src/code/z_view.c

```diff
@@ -51,80 +51,88 @@ void View_Init(View* view, GraphicsContext* gfxCtx) {
     view->up.z = 0.0f;
     view->eye.z = -1.0f;
 
-    if (D_8012ABF0) {
-        if (D_8012ABF0 == 0) {}
+    if (sViewNotInitialized) {
+        if (sViewNotInitialized == 0) {}
         osSyncPrintf("\nview: initialize ---\n");
-        D_8012ABF0 = false;
+        sViewNotInitialized = false;
     }
 
     view->unk_124 = 0;
-    view->flags = 1 | 2 | 4;
+    view->dirtyFlags = VIEW_VIEWING | VIEW_VIEWPORT | VIEW_PROJECTION_PERSPECTIVE;
     View_InitDistortion(view);
 }
 
-void func_800AA358(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up) {
+void View_LookAt(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up) {
     if (eye->x == lookAt->x && eye->z == lookAt->z) {
         eye->x += 0.1f;
     }
 
     view->eye = *eye;
     view->lookAt = *lookAt;
     view->up = *up;
-    view->flags |= 1;
+    view->dirtyFlags |= VIEW_VIEWING;
 }
 
-void func_800AA3F0(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up) {
+/*
+ * Unused. View_LookAt is always used instead. This version is similar but
+ * is missing the input sanitization and the update to the dirtyFlags.
+ */
+void View_LookAtUnsafe(View* view, Vec3f* eye, Vec3f* lookAt, Vec3f* up) {
     view->eye = *eye;
     view->lookAt = *lookAt;
     view->up = *up;
```

Comment:
```suggestion
    view->eye = *eye;
    view->at = *at;
    view->up = *up;
```
As an fyi, camera uses the term `at` and "viewing" should be made consistent. We've had the discussion before about `at` vs `lookAt` and concluded that we should be using `at`, but of course that conversation can always be opened again. But it should be consistent.

Reply:
I agree, I also like `at` for the variable. Seems like the most common name for the eye/at/up triple, and is also the name used for the parameter to `guLookAt`.

Reply:
9b79cf4ca